src/routes/content/[...id].svelte

```diff
@@ -0,0 +1,50 @@
+<script lang="ts">
+	import { goto } from "$app/navigation";
+	import type * as api from "src/api";
+	import { InsertFormType } from "src/api/forms/base";
+	import Content from "src/components/info/Content.svelte";
+	import DeleteModal from "src/components/info/DeleteModal.svelte";
+	import { draftForm } from "src/utils/forms/stores";
+	import { v4 as uuid } from "uuid";
+
+	export let content: api.Content;
+
+	function handleEdit() {
+		const id = uuid();
+		draftForm.createForm(id, [{
+			type: InsertFormType.CONTENT,
+			value: content,
+		}]);
+
+		goto(`/forms/${id}`);
+
+		// TODO: handle updates to existing nodes within insertForm functionality to support editing content
+	}
```

Comment:
Yep. IIRC what we talked about was to first create a JSONified copy of the existing content, save it as a new row in the revision table, confirm that the revision was successfully inserted, and then replace all fields on the target node with the current form values. Yes?

Reply:
Oh, right, yes. This comment is out of date now, it seems I forgot to remove it after implementing that. All of those things has been implemented in this PR.

Reply:
I'd thought so, but it's late and I thought maybe I missed it... : 1

Shall we just delete the comment, then?